discovery-infra/test_infra/kubeapi_utils.py

```diff
@@ -91,3 +98,45 @@ def get_hostname_for_agent(agent, nodes_details):
             if interface['macAddress'].lower() == mac_address:
                 return node_metadata['name']
 
+
+def delete_kube_api_resources_for_namespace(
+        kube_api_client,
+        name,
+        namespace,
+        **kwargs
```

Comment:
if not using it explicitly, then just replace with ``secret_name=None, installenv_name=None,...``